source/introduction.rst

```diff
@@ -26,18 +27,21 @@ Perun Protocol
 
 The project is based on a on the perun protocol, a set of cryptographic
 protocols invented and formally analyzed by cryptographic researchers at TUDa
-and the University of Warsaw. Perun protocol allows users to shift transaction
-and smart contract execution away from the blockchain into so-called payment
-and state-channels. These channels are created by locking coins on the
-blockchain and can be updated directly between the users and without any
-on-chain interaction. This makes state-channel-based transactions much faster
-and cheaper than on-chain transactions. The underlying blockchain guarantees
-that all off-chain transactions will be enforced on-chain eventually. In
-comparison to other channel technologies like the Lightning Network, the Perun
-construction offers the following unique features:
-
-
-Perun’s state-channel virtualization
+and the University of Warsaw. Perun protocol allows users to shift
+transaction and smart contract execution away from the blockchain into
+so-called payment and state-channels. These channels are created by locking
+coins on the blockchain and can be updated directly between the users and
+without any on-chain interaction. This makes state-channel-based transactions
+much faster and cheaper than on-chain transactions. The underlying blockchain
+guarantees that all off-chain transactions will be enforced on-chain
+eventually. For a detailed explanation on the concept of state channels,
+please read section on :ref:`State Channels`.
```

Comment:
link not working

Reply:
I am not sure if the link will work on github, but on my local machine I tried and it works. Can you provide more info here on when it does not work ? @anagha-ks

Reply:
Yea, this will not work on github, but works on sphinx page.